resources/build/webpack.development.js

```diff
@@ -153,6 +155,22 @@ module.exports = {
         ],
       },
 
+      /**
+       * Handle SVG sprites.
+       */
+      {
+        test: utils.tests.svgs,
+        use: [
+          {
+            loader: 'svg-sprite-loader',
+            options: {
+              extract: true,
+              spriteFilename: 'images/sprite.svg',
```

Comment:
Extract mode should not be used in webpack development/hot mode in order to have the svg sprite load correctly.
```suggestion
              extract: false,
```